### ISSUE TITLE
Standards shouldn't encourage colored buttons with black text.

### ISSUE THREAD

**Issue by StraightfaceStudios** [18F/web-design-standards#765]
I would discourage having any colored buttons with black text.  If you're going to use a color for a button's background it's best to go with white text and choose an appropriate background color.

I would argue that #02bfe7 + black text is not an easy to view combination.   Anything with 99% saturation is harsh on the eyes and difficult to read as a background for text for the same reason that black text on yellow backgrounds is strongly discouraged due to eye strain.    For instance the "Download The Design Files" button on this page (https://playbook.cio.gov/designstandards/visual-style/) is harsh on the eyes. 

If you desaturate the alt color + black you get a luminance shift (in sRGB gamma) of 45%.  When you compare that to the Primary Color as the background vs White you end up with a 63% tonal shift (in sRGB gamma).   That's substantially more perceptual contrast and is easier to discern. 

![image](https://cloud.githubusercontent.com/assets/2539036/10214412/d029c00c-67cc-11e5-989b-5539e8dbefb4.png)

**Comment by joshbruce** [comment]
@StraightfaceStudios - Would you say this is the same regarding background and text colors in general? I'm thinking about the "Visual style" page - at the bottom regarding contrast and accessibility.

**Comment by nickbristow** [comment]
While I usually just look at contrast for compliance reasons, color/brightness ratio is also a good metric to look at and you make a good point @StraightfaceStudios 

I wouldn't fail it from a 508 perspective, but I'm all for making things better @juliaelman

**Comment by juliaelman** [comment]
@nickbristow thank you for confirming and to @StraightfaceStudios for bringing this to our attention! We'll be sure to add this into our workflow processes and make any adjustments accordingly.

**Comment by msecret** [comment]
I believe we've removed all instances of colored buttons with black text, so I will close this issue. Please feel free to re-open it if this is incorrect.
